packages/react-storage/src/components/StorageBrowser/views/LocationActionView/CopyView/useCopyView.ts

```diff
@@ -27,39 +36,20 @@ export const useCopyView = ({
 
   const getInput = useGetActionInput();
 
-  const [{ tasks }, handleProcess] = useProcessTasks(
+  const [processState, handleProcess] = useProcessTasks(
     copyHandler,
-    // @ts-ignore
-    // temporarily ignore error that items is missing
-    // until PR that refactors useProcessTasks is merged
     fileDataItems,
-    {
-      concurrency: 1,
-    }
+    { concurrency: 4 }
```

Comment:
Is there a rationale for choosing this number?

Reply:
Yes, it's highest number of concurrent tasks we want to run. Cen be moved to a constant with comments in a follow up